src/main/java/ch/njol/skript/lang/util/CodeSection.java

```diff
@@ -0,0 +1,68 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.lang.util;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.lang.Section;
+import ch.njol.skript.lang.TriggerSection;
+import ch.njol.skript.lang.parser.ParserInstance;
+import ch.njol.util.Kleenean;
+
+import java.util.List;
+
+/**
+ * A {@link Section} with code contents.
+ */
+public abstract class CodeSection extends Section {
```

Comment:
I feel like this class does not serve many purposes? I don't seem to understand why this inheritance is needed, instead of just adding it to the Section class instead. Could you provide me with some details perhaps?

Reply:
Originally the section was meant to automatically load the code in init, but I changed that behaviour later, so I'll remove the class now